pandas/io/sql.py

```diff
@@ -478,6 +478,41 @@ def read_sql(
     -------
     DataFrame or Iterator[DataFrame]
 
+    Examples
+    --------
+    Read data from SQL via either a SQL tablename or a SQL query
+
+    >>> pd.read_sql('table_name', 'postgres:///db_name')  # doctest:+SKIP
```

Comment:
Maybe the examples can demonstrate connecting to a sqlite db so we don't have to skip all these doctests

Reply:
Done